eng/pipelines/common/global-build-job.yml

```diff
@@ -58,15 +67,6 @@ jobs:
           df -h
         displayName: Disk Usage after Build
 
-    - task: PublishBuildArtifacts@1
```

Comment:
@joperezr moved this to be after the extraStepsTemplate expansion to include the logs for those steps.

Reply:
Nice, I should've thought of that 😄